toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -509,12 +509,8 @@ chroot_and_install_rpms libtirpc
 chroot_and_install_rpms rpcsvc-proto
 build_rpm_in_chroot_no_install libnsl2
 
-build_rpm_in_chroot_no_install finger
-
 # tcp_wrappers needs libnsl2, finger
```

Comment:
Could you also update the comment?

Reply:
Also, please mention that `libns12` **maybe** can be removed now. Similar to the way other comments do this below. It's our way of tracking what more we can remove from the toolchain when we have a free minute.:)